tests/Routing/RoutingRouteTest.php

```diff
@@ -1484,13 +1484,38 @@ public function testRouteRedirect()
         $router->get('contact_us', function () {
             throw new \Exception('Route should not be reachable.');
         });
-        $router->redirect('contact_us', 'contact', 302);
+        $router->redirect('contact_us', 'contact');
```

Comment:
No need to change this, you could just add a new test in order to test the new functionality.

Reply:
This test tests that the default redirect is a 302 and new tests have been added to test for a custom redirect code (e.g 301) as well as the new permanentRedirect method.

Reply:
I know but as a rule of thumb, never change tests if not necessary. You could just add new tests and the existing ones will still work.